gravitee-apim-e2e/ui-test/integration/apim/ui/apis/ui-api-notifications.spec.ts

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ADMIN_USER, API_PUBLISHER_USER } from '@fakers/users/users';
+import { ApisFaker } from '@gravitee/fixtures/management/ApisFaker';
+import { ApiImport } from '@model/api-imports';
+import apiDetails from 'ui-test/support/PageObjects/Apis/ApiDetails';
+import { PlanStatus } from '../../../../../lib/management-webclient-sdk/src/lib/models';
+import { PlansFaker } from '@gravitee/fixtures/management/PlansFaker';
+
+let api: ApiImport;
+
+describe('Notifications page', () => {
+  beforeEach(() => {
+    cy.loginInAPIM(ADMIN_USER.username, ADMIN_USER.password);
+    cy.visit(`/#!/environments/default/apis/${api.id}/notification-settings`);
+    cy.contains('Portal Notifications').should('be.visible');
+  });
```

Comment:
Would put it after the `before` one, just to ease the reading

Reply:
My fault, means I need to make quick change to info and plans tests too 😅